release/gcb/generate_manifest.sh

```diff
@@ -221,6 +227,7 @@ pushd "${CLONE_DIR}"
   MANIFEST_FILE="$PWD/manifest.txt"
 
   git clone "https://github.com/${CB_GITHUB_ORG}/istio" -b "${CB_BRANCH}"
+  git clone "https://github.com/${CB_GITHUB_ORG}/cni" -b "${CB_BRANCH}"
```

Comment:
does this mean we need to make release-x.y.z branches in istio/cni?

Reply:
I have the same question, and I suspect we should.

Reply:
Per above - it seems like we are going around in circles a bit on how to do this.  Probably need a discussion on it.

Reply:
I think the plan should be that when we run the release scripts out of istio/istio it will create a tag in the CNI repo similar to what is done for the API and proxy repos.